src/ui/App.css

```diff
@@ -107,11 +107,10 @@ select:focus {
   width: 100%;
   height: 160px;
 
-  backdrop-filter: blur(10px);
```

Comment:
Keep the backdrop blur.

Consider the visibility of this:
![image](https://github.com/Grasscutters/Cultivation/assets/25207995/5dcdfffc-9c67-409e-b27a-69d3fa537bc0)

vs. this:
![image](https://github.com/Grasscutters/Cultivation/assets/25207995/f2a227bd-5264-4916-9d76-6c98171655bf)

"Connect to Grasscutter" can be slightly more difficult to see. I love vague dark gradients as much as the next guy, but sacrificing visibility (even if minor) unnecessarily is not needed. The blur has an added bonus of improving visibility regardless of which background is chosen, so if there is an even *more* busy background to come out in a future update, there is no risk of visual clashing.